test/lib/ansible_test/_util/target/setup/bootstrap.sh

```diff
@@ -91,6 +91,7 @@ bootstrap_remote_alpine()
         python3-dev
         ${py_pkg_prefix}-pip
         sudo
+		crun
```

Comment:
If this works, it should be added here instead (so it's specific to the test using it):

https://github.com/ansible/ansible/blob/a7d2a4e03209cff1e97e59fd54bb2b05fdbdbec6/test/integration/targets/ansible-test-container/runme.py#L1053